pyveda/vedaset/stream/vedastream.py

```diff
@@ -146,6 +146,13 @@ def __init__(self, mltype, classes, count, gen, partition, bufsize,
         if auto_startup:
             self._start_consumer()
 
+    @classmethod
+    def from_vc(cls, vc):
+        return VedaStream(mltype = vc.mltype, classes = vc.classes, count = vc.count,
```

Comment:
No problem calling and returning `VedaStream` here, but if in the future we wanted to do something like subclass VedaStream, `class CoolerVedaStream(VedaStream): ...` and we called `.from_vc()` on it, we'd only get back a `VedaStream`.  Since you defined a classmethod here, you have access to whatever the class is itself, which is what `cls` is, so instead we can just return an instance of whatever the `cls` is that's calling the method: `return cls(vc.mltype, ...)`